app/config.py

```diff
@@ -456,6 +459,7 @@ class Development(Config):
 class Decoupled(Development):
     NOTIFY_ENVIRONMENT = "decoupled"
     ADMIN_BASE_URL = "http://admin.ecs.local:6012"
+    ADMIN_EXTERNAL_URL = "https://admin.development.emergency-alerts.service.gov.uk"
```

Comment:
Should we be hardcoding urls?

Reply:
Urls are now constructed from a base string and an environment variable